docker-compose.yml

```
@@ -0,0 +1,59 @@
+version: "3.8"
+
+services:
+  charterafrica:
+    depends_on:
+      mongodb_container:
+        condition: service_healthy
+    build:
+      context: .
+      dockerfile: Dockerfile.charterafrica
+      args:
+        MONGO_URL: mongodb://root:rootpassword@host.docker.internal:27017/charterafrica?authSource=admin
+        PAYLOAD_SECRET_KEY: ${PAYLOAD_SECRET_KEY}
+        SENTRY_AUTH_TOKEN: ${SENTRY_AUTH_TOKEN}
+        SENTRY_ORG: ${SENTRY_ORG}
+        NEXT_PUBLIC_SENTRY_DSN: ${CHARTERAFRICA_SENTRY_DSN}
+        SENTRY_ENV: ${SENTRY_ENV}
+        SENTRY_PROJECT: ${SENTRY_PROJECT}
+    ports:
+      - 3000:3000
+    container_name: charterafrica
+    env_file:
+      - apps/charterafrica/.env.local
+    networks:
+      - net
+
+  mongodb_container:
+    container_name: mongodb_container
+    image: mongo:latest
```

```suggestion
    image: mongo:6.0
```
Or `mongo:6.0.13` so that we all use the exact same version